wqflask/tests/wqflask/show_trait/test_export_trait_data.py

```diff
@@ -0,0 +1,92 @@
+import unittest
+from wqflask.show_trait.export_trait_data import dict_to_sorted_list
+from wqflask.show_trait.export_trait_data import cmp_samples
+
+
+class TestExportTraits(unittest.TestCase):
+	"""Test methods related converting dict to sortedlist"""
+    def test_dict_to_sortedlist(self):
+        '''Test for conversion of dict to sorted list'''
```

Comment:
**issue:** Please use proper double quotes instead of single quotes for docstrings.